pt-BR/action_controller_overview.md

```diff
@@ -901,21 +901,21 @@ class ClientsController < ApplicationController
 end
 ```
 
-This will read and stream the file 4kB at the time, avoiding loading the entire file into memory at once. You can turn off streaming with the `:stream` option or adjust the block size with the `:buffer_size` option.
+Esse método irá ler e transmitir o arquivo 4kb por vez, evitando carregar o arquivo inteiro em memória de uma vez. Você pode desativar a transmissão com a opção `:stream` ou ajustar o tamanho do bloco com a opção `:buffer_size`.
 
-If `:type` is not specified, it will be guessed from the file extension specified in `:filename`. If the content type is not registered for the extension, `application/octet-stream` will be used.
+Se a opção `:type` não for especificada, será presumido de acordo com a extensão especificada no `:filename`. Se o tipo do conteúdo (`content_type`) não estiver registrado para a extensão, será usado `application/octet-stream`.
 
-WARNING: Be careful when using data coming from the client (params, cookies, etc.) to locate the file on disk, as this is a security risk that might allow someone to gain access to files they are not meant to.
+WARNING:  Tenha cuidado ao utilizar dados vindos do _browser_ ( _params_, _cookies_, etc.) para localizar um arquivo no disco, pois é um risco de segurança que pode permitir alguém a ter acesso a arquivos não permitidos.
 
-TIP: It is not recommended that you stream static files through Rails if you can instead keep them in a public folder on your web server. It is much more efficient to let the user download the file directly using Apache or another web server, keeping the request from unnecessarily going through the whole Rails stack.
+TIP: Não é recomendado que você transmita arquivos estáticos através do _Rails_, você pode ao invés mante-los em uma pasta publica no seu servidor _web_. É muito mais eficiente deixar os usuários baixarem os arquivos diretamente utilizando _Apache_ ou outro servidor _web_, deixando a requisição desnecessariamente passar por todo o fluxo do _Rails_. 
```

Comment:
Nos outros documentos, nós não marcamos `Rails` em itálico ou sublinhado. Vocês querem começar a deixar desta forma? Eu não vejo muita vantagem em fazer isso, considerando que o Rails é o tema central do guia todo.

Reply:
Boa, vou tirar o itálico!